client/src/components/Donation/DonationOptionsText.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const DonationOptionsText = ({ isAlert }) => {
+  const Title =
+    'Want to make a bigger one-time donation, mail us a check, or give in other ways?';
+
+  return (
+    <>
+      {!isAlert && (
```

Comment:
Can we make this declarative? It's OK for a spike but can quickly become a source for bugs

Reply:
Thanks, it should be declarative now.